### PR TITLE
Internationalized static code through the app (#1wut862)

### PR DESCRIPTION
**Pull request by azkyakhan**

### CODE REVIEW

File: src/views/Completed.vue [hunk 1]
```diff
@@ -2,9 +2,9 @@
   <ion-page>
     <ion-header :translucent="true">
       <ion-toolbar>
-        <ion-title>10 to 26 orders</ion-title>
+        <ion-title>10 to 26 {{ $t("orders") }}</ion-title>
```
Comment (adityasharma7) [inline src/views/Completed.vue:5]: to should also be internationalized.

File: src/views/Completed.vue [hunk 2]
```diff
@@ -191,9 +190,9 @@ export default defineComponent({
     async shipOrderAlert() {
       const alert = await alertController
         .create({
-           header: 'Ship orders',
-           message: 'You are shipping 15 orders. <br> You cannot unpack and edit orders after they have been  shipped. Are you sure you are ready to ship this orders.',       
-           buttons: ['Cancel', 'Ship'],
+           header: this. $t("Ship orders"),
+           message: this.$t("You are shipping 15 orders. You cannot unpack and edit orders after they have been  shipped. Are you sure you are ready to ship this orders."),
```
Comment (adityasharma7) [inline src/views/Completed.vue:194]: We have removed the line break. Is it not required?

File: src/views/EditPackagingModal.vue [hunk 3]
```diff
@@ -54,24 +54,24 @@
 
         <div class="product-metadata">
           <ion-item lines="none">   
-            <ion-label>Select box</ion-label>
+            <ion-label>{{ $t("Select box") }}</ion-label>
             <ion-select>
-              <ion-select-option>Box A  Type3</ion-select-option>
-              <ion-select-option>Box B  Type2</ion-select-option>
+              <ion-select-option>{{ $t("Box A") }} {{ $t("Type 3") }}</ion-select-option>
```
Comment (adityasharma7) [inline src/views/EditPackagingModal.vue:59]: I am not sure if this is to be internationalized. I think these values will dynamic. Please verify.

Comment (adityasharma7) [reply src/views/EditPackagingModal.vue:59]: Check for everywhere we either taking inputs or showing info for boxes

File: src/views/Settings.vue [hunk 4]
```diff
@@ -2,52 +2,52 @@
...
       </ion-card>
 
       <ion-card>
         <ion-item lines="none">
-          <ion-label>Fulfillment: On</ion-label>
+          <ion-label>{{ $t("Fulfillment") }} : {{ $t("On") }}</ion-label>
         </ion-item>
         <ion-item lines="none">
-          <ion-label class="text-wrap">Broadway has 47 outstanding orders and 77 in progress orders.</ion-label>
+          <ion-label class="text-wrap">{{ $t("Broadway has 47 outstanding orders and 77 in progress orders.") }}</ion-label>
```
Comment (adityasharma7) [inline src/views/Settings.vue:37]: This message has dynamic values

File: src/views/OpenOrders.vue [hunk 5]
```diff
@@ -18,34 +18,34 @@
         <ion-item lines="none">
           <ion-checkbox slot="start"/>
           <ion-label>
-            Same Day
-            <p>37 orders, 40 items</p>
+            {{ $t("Same Day") }}
```
Comment (adityasharma7) [inline src/views/OpenOrders.vue:21]: I think this should be dynamic value

File: src/views/InProgress.vue [hunk 6]
```diff
@@ -183,31 +183,31 @@ export default defineComponent({
...
       return alert.present();
     },
     async reportIssueAlert() {
       const alert = await alertController
         .create({
-          header: 'Report an Issue',
-          message: 'WJ06-XL-Purple, and 5 other products are identified as unfulfillable.<br> 4 other orders  containing these products will be  unassigned  from this store and sent to be rebrokered.',       
-          buttons: ['Cancel', 'Report'],
+          header: this.$t("Report an Issue"),
+          message: this.$t("WJ06-XL-Purple, and 5 other products are identified as unfulfillable. 4 other orders  containing these products will be  unassigned  from this store and sent to be rebrokered."),
```
Comment (adityasharma7) [inline src/views/InProgress.vue:209]: I think the message has dynamic values, WJ06-XL-Purple is an SKU

File: src/views/InProgress.vue [hunk 7]
```diff
@@ -76,42 +76,42 @@
...
-                    <ion-select-option value="box2">Box B Type 2</ion-select-option>
+                    <ion-select-option value="box1">{{ $t("Box A") }} {{ $t("Type 3") }}</ion-select-option>
+                    <ion-select-option value="box2">{{ $t("Box B") }} {{ $t("Type 2")}}</ion-select-option>
                   </ion-select>      
                 </ion-item>
               </div>
               <div v-if="segment == 'issue'">
                 <ion-item lines="none">  
-                  <ion-label>Select Issue</ion-label>  
+                  <ion-label>{{ $t("Select issue") }}</ion-label>
```
Comment (adityasharma7) [inline src/views/InProgress.vue:97]: Need to verify if this will be internationalised

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: src/views/Completed.vue (hunk 1 above)
Comment (azkyakhan) [reply src/views/Completed.vue:5]: Ok sir

File: src/views/EditPackagingModal.vue (hunk 3 above)
Comment (dt2patel) [reply src/views/EditPackagingModal.vue:59]: this will be dynamic, no need to internationalize

File: src/views/Completed.vue (hunk 2 above)
Comment (azkyakhan) [reply src/views/Completed.vue]: It is required sir. I have added it again.

File: src/views/EditPackagingModal.vue (hunk 3 above)
Comment (azkyakhan) [reply src/views/EditPackagingModal.vue]: Ok sir

File: src/views/Settings.vue (hunk 4 above)
Comment (azkyakhan) [reply src/views/Settings.vue]: Ok

File: src/views/OpenOrders.vue (hunk 5 above)
Comment (azkyakhan) [reply src/views/OpenOrders.vue]: Ok

File: src/views/InProgress.vue (hunk 6 above)
Comment (azkyakhan) [reply src/views/InProgress.vue]: I have implemented dynamic internationalization here.

File: src/views/InProgress.vue (hunk 7 above)
Comment (azkyakhan) [reply src/views/InProgress.vue:97]: Ok sir

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**